### PR TITLE
Add OpenShift service scope

### PR DESCRIPTION
**Pull request by suehighmoor**

### CODE REVIEW

File: articles/openshift/oshift-sco.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+- Kubernetes 1.11
+
+- UKCloud for OpenStack is the underlying Infrastructure-as-a-Service (IaaS) platform
+
+## Default setup
+
+UKCloud for OpenShift is delivered as a private Platform-as-a-Service (PaaS).
+
+Each customer will be provided with an isolated OpenShift cluster within which they can build their containerized solutions.
```
Comment (SteveHallUKCLoud) [inline articles/openshift/oshift-sco.md]: containerized or containerised? article has both spellings

File: articles/openshift/oshift-sco.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+> [!NOTE]
+> An operational OpenShift cluster must consist of a minimum of a Foundation Pack plus a single Runtime Pack, however we strongly recommend the use of two or more Runtime Packs for application resilience.
+
+### Foundation Pack
+
+Provides the foundation OpenShift services to support the orchestration and management of the container runtime environments and include certain [optional extras](#optional-extras) free of charge as defined below. Each customer environment requires only a single Foundation Pack.
+
+### Runtime Pack
+
+Provides the footprint to host customer’s containerised applications and services. Customers can choose to mix and match their Runtime Packs, and Runtime Packs can be called off individually, however UKCloud strongly recommends the initial deployment of two or more Runtime Packs for production environments to provide resilience to your workloads in the event of the failure of a single Runtime Pack.
```
Comment (SteveHallUKCLoud) [inline articles/openshift/oshift-sco.md]: or containerized

File: articles/openshift/oshift-sco.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+### Additional networks
+
+By default, your OpenShift cluster will only be connected to the internet. Your OpenShift environment can be provisioned on other networks such as HSCN, Janet, RLI and PSN at a flat fee month per network plus the additional network charges.
+
+### Registry storage
+
+A benefit of OpenShift is its ability to build Docker images directly from your source code, deploy them, and manage their lifecycle. To enable this, OpenShift utilises an integrated Docker registry to locally manage images. This registry is stored securely outside of your OpenShift environment to help ensure the resilience and availability of your images.
+
+UKCloud provides you with an initial 60GB of registry storage as part of your Foundation Pack; with any additional registry storage being chargeable.
```
Comment (SteveHallUKCLoud) [inline articles/openshift/oshift-sco.md]: GiB?

File: articles/openshift/oshift-sco.md (hunk 1 above)
Comment (stevedicko) [reply articles/openshift/oshift-sco.md]: Should be containerised... bloody spellcheck!

File: articles/openshift/oshift-sco.md (hunk 2 above)
Comment (stevedicko) [reply articles/openshift/oshift-sco.md]: containerised

File: articles/openshift/oshift-sco.md (hunk 3 above)
Comment (stevedicko) [reply articles/openshift/oshift-sco.md]: Yep GiB

**Pull request merged**